post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -21,8 +21,7 @@ include::modules/checking-mco-status.adoc[leveloffset=+2]
 [id="using-machineconfigs-to-change-machines"]
 == Using `MachineConfig` objects to configure nodes
 
-Tasks in this section let you create `MachineConfig` objects to modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with machine configs, see
- content related to link:https://access.redhat.com/solutions/5307301[changing MTU network settings], link:https://access.redhat.com/solutions/5096731[adding] or
+Tasks in this section let you create `MachineConfig` objects to modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with machine configs, see content related to link:https://access.redhat.com/solutions/5096731[adding] or
```

Comment:
s/Tasks in this section let you create `MachineConfig` objects to modify files/You can use the tasks in this section to create `MachineConfig` objects that modify files

From ISG:
**let** _v_
Avoid stating that inanimate objects grant abilities to people, as in “the product lets you...” Whenever
possible, use a direct, user-focused alternative such as “you can use the product to...” or “with this product,
users can...”